src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1928,6 +1935,7 @@ protected override TScalarPredictor TrainCore(IChannel ch, RoleMappedData data,
             var pred = new LinearBinaryPredictor(Host, in maybeSparseWeights, bias);
             if (!(_loss is LogLoss))
                 return pred;
+            //REVIEW: does this need to be set to Platt, or the Calibrator in the args? The calibrator in the ars is not used anywhere else.
             return new ParameterMixingCalibratedPredictor(Host, pred, new PlattCalibrator(Host, -1, 0));
```

Comment:
>new PlattCalibrator(Host, -1, 0) [](start = 70, length = 32)

is this correct? The Calibrator in the arguments is not used.

Reply:
`Platt` is a standard option for SVM. I don't know what other calibrators are in ML.NET, but if they also map a scalar to probability, we should be able to them as well.

---
In reply to: [239916963](https://github.com/dotnet/machinelearning/pull/1816#discussion_r239916963) [](ancestors = 239916963)

Reply:
I'll log an issue about this. 

---
In reply to: [240730650](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240730650) [](ancestors = 240730650,239916963)

Reply:
Sure.

---
In reply to: [240733299](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240733299) [](ancestors = 240733299,240730650,239916963)